pkg/ottl/ottlfuncs/README.md

```diff
@@ -877,6 +878,26 @@ Examples:
 
 - `Int(Log(attributes["duration_ms"])`
 
+### MD5
+
+`MD5(value)`
+
+The `MD5` Converter converts the `value` to a md5 hash/digest.
+
+The returned type is string.
+
+`value` is either a path expression to a string telemetry field or a literal string. If `value` is another type an error is returned.
```

Comment:
Do we have to limit the input to strings? Could we include numbers and byte slices as well for starters?

Reply:
I took a second look and all our other hash functions have the same restriction, let's just leave this for now.